config/rbac/role.yaml

```diff
@@ -41,6 +38,17 @@ rules:
   - list
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - services
```

Comment:
nit: looks like you could move `services` into the rule just above with `namespaces`, etc., if you want to keep similar things together.